src/main/java/seedu/exercise/logic/commands/AddCommand.java

```diff
@@ -0,0 +1,68 @@
+package seedu.exercise.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.exercise.logic.parser.CliSyntax.PREFIX_CALORIES;
+import static seedu.exercise.logic.parser.CliSyntax.PREFIX_DATE;
+import static seedu.exercise.logic.parser.CliSyntax.PREFIX_MUSCLE;
+import static seedu.exercise.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.exercise.logic.parser.CliSyntax.PREFIX_QUANTITY;
+import static seedu.exercise.logic.parser.CliSyntax.PREFIX_UNIT;
+
+import seedu.exercise.logic.commands.exceptions.CommandException;
+import seedu.exercise.model.Model;
+import seedu.exercise.model.exercise.Exercise;
+
+/**
+ * Adds an exercise to the exercise book.
+ */
+public class AddCommand extends Command {
+
+    public static final String COMMAND_WORD = "add";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a exercise to the exercise list. "
+        + "Parameters: "
+        + PREFIX_NAME + "EXERCISE NAME "
+        + PREFIX_DATE + "DATE "
+        + PREFIX_CALORIES + "CALORIES "
+        + PREFIX_QUANTITY + "QUANTITY "
+        + "[" + PREFIX_MUSCLE + "MUSCLE]...\n"
```

Comment:
The message printed out to feedback is lacking a units. Please add in `PREFIX_UNIT + "UNITS "`

Reply:
done